StringCalculatorKata/StringCalculator.cs

```diff
@@ -16,9 +16,31 @@ private int Parser(string s)
         private void CalculateSum(string[] Numbers, out int Sum)
         {
             Sum = 0;
-            for(int i = 0; i < Numbers.Length; i++)
+            string[] Negatives = { };
+            bool HasNegatives = false;
+
+            for (int i = 0; i < Numbers.Length; i++)
+            {
+                int Number = Parser(Numbers[i]);
+                if (Number < 0)
+                {
+                    HasNegatives = true;
```

Comment:
how could I get rid of this variable? (is there something else that I could use to determine if there were negatives or not?)